Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -558,6 +558,20 @@
       components:
       - SurplusBundle
 
+- type: listing
+  id: uplinkBananaCreamPie
+  name: uplink-banana-cream-pie-name
+  description: uplink-banana-cream-pie-desc
+  productEntity: FoodPieBananaCream
+  cost:
+    Telecrystal: 1
+  categories:
+  - UplinkJob
+  conditions:
+  - !type:BuyerJobCondition
+    whitelist:
+    - Clown
+
```

Comment:
this should probably be blacklisted from surplus

Reply:
well, gatfruit seeds are not blacklisted from surplus (as they are not uplink exclusive unlike Necronomicon), so I dunno

Reply:
The Necronomicon is black listed because other jobs cannot use it. Similarly, a traitor engineer can’t really do much with a cream pie he got from his surplus.

Reply:
you can use it to stun people and drop them weapons